### ISSUE TITLE
Serve zoeappleseed.com from github

### ISSUE THREAD

**Issue by JAForbes** [ZoeAppleseed/zoeappleseed.github.io#9]
Automated updating of the main site based on the `gh-pages` branch will save a lot of effort.

**Comment by JAForbes** [comment]
I've transferred ownership of the website to @ZoeAppleseed and changed the repo named to `zoeappleseed.github.io` so github will serve it as a webpage automagically.

I've also set up all the CNAME and A record stuff at Dreamhost, but it could take a day or two for that to propagate.

I've added a `console.log` saying `"Hosted at zoeappleseed/zoeappleseed.github.io"` so it will be easy to know when we are live.

When we are live.  I'll

- [ ] Remove the "hosted" at log
- [ ] Remove the json `$.get` from the github api, and make it just a local get from this repo.


After I close this issue, I can move on to adding the etsy link #10
